go-packages/meep-net-char-mgr/algo-segment.go

```diff
@@ -387,15 +387,15 @@ func (algo *SegmentAlgorithm) createMetricsEntry(srcElem string, dstElem string)
 	creationTime["creationTime"] = time.Now()
 
 	// Entries are created with no values, sidecar will only fill them, otherwise, won't be cleared
-	_ = algo.rc.SetEntry(algo.BaseKey+dstElem+":"+srcElem, creationTime)
+	// _ = algo.rc.SetEntry(algo.BaseKey+dstElem+":"+srcElem, creationTime)
```

Comment:
why commenting the creation of the entries?

Reply:
TC sidecars now use the metric-store package to set these entries in the redis DB. The net char manager no longer needs to create the entries for the sidecars.

Reply:
I will remove the commented line.